### ISSUE TITLE
Wo Daten bearbeiten

### ISSUE THREAD

**Issue by MaddogMccree** [iobroker-community-adapters/ioBroker.statistics#90]
Hallo

Ich nutze Statistics um DAten von einem esp32 AI on the Edge Wasserzähler
zu loggen und auszuwerten.
Jetzt hat der esp leider falsche werte aufgezeichnet und ich sollte diese wieder ändern 
da sonst alles nicht mehr stimmt.
Wenn ich nun in den Objecten die werte ändere ( in Temp und auch in Save ) werden Sie 
nach einer gewissen Zeit wieder mit den alten falschen werte überschrieben.
Wo holt Statistics die Werte her und wo kann ich die ändern?
Und wenn alles nichts hilft was könnte Ich amachen um alles von neuem aufzuzeichnen ohne statistics zu löschen?

Viele Grüße
Michael Dreher

**Comment by foxthefox** [comment]
Was genau wird mit welcher Art von statistic geloggt?
Und was soll dann manipuliert werden?

**Comment by MaddogMccree** [comment]
Hallo

Ich habe ein AI on the Edge esp32 modul das mir die Wasseruhr trackt.
Der "Value" wert lass ich über Statistics tracken mit "Averange Value" / Counter in Delta Cosumption / Ignore Errors
und in die Gruppe Wasser Total mit Counterfactor 1
Der ESP hat nun 217.334 m³ ......... usw. geloggt
dann hatte er Probleme beim erkennen und hat den Wert 227.367 geloggt
nun hab ich in Statistics einen Wasserverbrauch innerhalb eines Messzyklus von 1000 m³ und somit verfälscht das mir die korrekte Berechnung.
Nun habe ich versucht unter Temp bei Month / Quarter / Year die korreckten Werte manuell einzutragen was aber nicht fiunktioniert da er irgendwoher den vorherigen Wert wieder dort einmträgt.
Nun wollte ich wissen wo ich diese ändern kann.

**Comment by foxthefox** [comment]
Ich schau heute mal

**Comment by klein0r** [comment]
> Wo holt Statistics die Werte her und wo kann ich die ändern?

Die Werte liegen im Arbeitsspeicher solange die Instanz läuft. Du müsstest die Instanz stoppen, den Wert in Temp anpassen und dann wieder starten.

**Comment by MecMax** [comment]
Hallo!
Ich nutze den Adapter erfolgreich für alle meine Stromzähler. Um das ganze noch etwas geschmeidiger bei Tarifänderungen zu machen, suche ich eine Möglichkeit, den Stromtarif in den Gruppen mit Javascript anzupassen. Habe ich da irgendeine Möglichkeit?

**Comment by klein0r** [comment]
> Habe ich da irgendeine Möglichkeit?

Ja, mit extendObject im JavaScript-Adapter. Aber wir sind hier bei Issues zum Adapter. Support / generelle Fragen bitte im Forum stellen

**Comment by mcm1957** [comment]
Und susserdem ist das Issue seit 2022 geschlossen ...
